packages/suite/src/storage/migrations/versions/25.12.0.ts

```diff
@@ -0,0 +1,22 @@
+import { createMigration } from '@suite/idb-migration-utils';
+
+import { SuiteDBSchema } from 'src/storage/definitions';
+
+export default createMigration<SuiteDBSchema>('25.12.0', (db, tx) => {
+    // @ts-expect-error `suiteSyncSettings` no longer exists, migrates to `suiteSyncSettings`
+    db.deleteObjectStore('labelingSettings');
```

Comment:
Open anonymous window, you'll see crash (currently migrations 25.11.0.1 and 25.12.0 create inconsitent state by default)
Must be wrapped in condition, because IDB functions are very unforgiving and not idempotent.

<img width="581" height="105" alt="Image" src="https://github.com/user-attachments/assets/7ae19c0b-745d-4285-a658-eeef4d676fd6" />

```suggestion
if(db.objectStoreNames.contains('labelingSettings'){
    db.deleteObjectStore('labelingSettings');
    }
```

Reply:
`db.objectStoreNames.contains('labelingSettings')` will throw TS error since the store was deleted